flink-contrib/flink-storm/src/main/java/org/apache/flink/storm/api/FlinkSubmitter.java

```diff
@@ -91,12 +91,11 @@ public static void submitTopology(final String name, final Map stormConf, final
 		final Configuration flinkConfig = GlobalConfiguration.loadConfiguration();
 		if (!stormConf.containsKey(Config.NIMBUS_HOST)) {
 			stormConf.put(Config.NIMBUS_HOST,
-					flinkConfig.getString(ConfigConstants.JOB_MANAGER_IPC_ADDRESS_KEY, "localhost"));
+					flinkConfig.getString(JobManagerOptions.ADDRESS, "localhost"));
 		}
 		if (!stormConf.containsKey(Config.NIMBUS_THRIFT_PORT)) {
 			stormConf.put(Config.NIMBUS_THRIFT_PORT,
-					new Integer(flinkConfig.getInteger(ConfigConstants.JOB_MANAGER_IPC_PORT_KEY,
```

Comment:
@zentol Thank you for your review. The default value of `JobManagerOptions.PORT` is 6123, I think maybe we use `flinkConfig.getInteger(JobManagerOptions.PORT)` here instead of `flinkConfig.getInteger(JobManagerOptions.PORT, 6123)` is better. What do you think? Thanks :)